### ISSUE TITLE
Multiple <dd> elements not rendered correctly

### ISSUE THREAD

**Issue by MarkCallow** [doxygen/doxygen#10863]
**Describe the bug**
When I have

```
<DL>
   <DT>Center
   <DT>Centre
   <DD> A point equidistant from all points
              on the surface of a sphere.
   <DD> In some field sports, the player who
              holds the middle position on the field, court,
              or forward line.
</DL>
```

in a doxygen comment an `Unexpected tag <dd> found` warning is issued and the content of the 2 tags is run together as one as seen in the screen shot below. The second screen shot shows how Firefox renders it.

The [HTML spec.](https://html.spec.whatwg.org/multipage/grouping-content.html#the-dd-element) shows in examples that multiple `<dd>` and `<dt>` tags are valid. Doxygen renders the multiple `<dt>` tags correctly.

**Screenshots**
<img width="1044" alt="Screenshot 2024-05-14 at 13 27 31" src="https://github.com/doxygen/doxygen/assets/2244683/e9cbe1aa-ee15-4bd6-a1ed-57ca03ea834a">

Rendering from Doxygen HTML output

<img width="710" alt="Screenshot 2024-05-14 at 13 32 04" src="https://github.com/doxygen/doxygen/assets/2244683/ebdad70a-2208-4567-9d5e-f396250b886c">

Firefox rendering of the example HTML.

**To Reproduce**

[issue10863.zip](https://github.com/doxygen/doxygen/files/15302805/issue10863.zip)

**Expected behavior**
No warning and each `<dd>` element rendered separately.

**Version**
Doxygen 1.9.8 on macOS Sonoma 14.4.1.

**Stack trace**
n/a

**Additional context**
n/a

**Comment by MarkCallow** [comment]
Thank you for the quick work, @albert-github. I know nothing about the Doxygen source code so am unable to give any opinion on the proposed patch.

**Comment by albert-github** [comment]
Code has been integrated in master on GitHub (please don't close the issue as this will be done at the moment of an official release).

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.11.0.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
